test/python/picongpu/.gitignore

```diff
@@ -1,2 +1,4 @@
 # generated by coverage.py
 .coverage
+LWFA
+warm_plasma
```

Comment:
why this gitignore here again, haven't you added them in the examples already?

Reply:
this ensure that the directories generated upon executing the the example test by hand are ignored by git, In contrast the example git ignore are used to ignore the directories generated when executing the examples from a copy or directly in PIConGPU.
The setup folder will always be generated at the call location by PyPIConGPU therefore we need to provide two different git ignore.

Reply:
discussed offline, current solution is fine